app/helpers/invitation_codes_helper.rb

```diff
@@ -6,13 +6,13 @@ def invite_hidden_tag(invite)
   end
 
   def invite_link(invite_code)
-    text_field_tag :invite_code, invite_code_url(invite_code), :readonly => true
+    text_field_tag :invite_code, invite_code_url(invite_code), :class => 'form-control', :readonly => true
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.